### ISSUE TITLE
Run Android App

### ISSUE THREAD

**Issue by kk8401** [kotarella1110/cordova-plugin-webpack#161]
Hi, 

I am trying to use the hotreload for my android device (Android 9).
I have follow the instruction and setup config.xml as described in the documentation.
When the app launched on my phone it is pointing to 10.0.2.2:8888

i have tried to setup host and port at devServer pointing to my laptop but doesn't seems to work.
  devServer: {
    host: '192.168.1.104',
    port: '8888'
  }, 

Thanks.

**Comment by kotarella1110** [comment]
@kk8401 Hi,  Is your laptop and your Android device in the same network environment (Wi-Fi)?

**Comment by kk8401** [comment]
yup same network. 
is there anything i need to config at my wifi?

**Comment by kotarella1110** [comment]
It works correctly in my environment.
This plugin will automatically get the local IP of the machine it is running on, so basically you don't need to fix webpack config.

**Comment by Kepro** [comment]
> When the app launched on my phone it is pointing to 10.0.2.2:8888

@kk8401 this IP 10.0.2.2 is your local LAN IP of your laptop or?

**Comment by peitschie** [comment]
10.0.2.2 happens to be loopback when running an Android emulator. Not entirely sure why it ends up being attempted here...

**Comment by peitschie** [comment]
Right... seems to be coming from here: https://github.com/kotarella1110/cordova-plugin-webpack/blob/master/src/webpackServe.ts#L78

For some reason, the `prepareUrls` is getting confused there
